packages/@dcl/sdk-commands/src/logic/get-free-port.ts

```diff
@@ -1,13 +1,13 @@
 import portfinder from 'portfinder'
 
-export async function previewPort() {
-  let resolvedPort = 0
+export async function getPort(port = 0, failoverPort = 2044) {
```

Comment:
Prefer no optionals or change all the occurrences. There is still logic like `args.port || await getPort()` which is inconsistent.

```suggestion
export async function getPort() {
```